deployment/nixops/server.nix

```diff
@@ -99,6 +103,12 @@
             expires off;
             '';
           };
+          "/tutorial/" = {
+            root = "${plutus.plutus-playground.documentation-site}/";
+          };
+          "/haddock/" = {
+            root = "${plutus.plutus-playground.documentation-site}/";
```

Comment:
This seems odd to me but it seems to work!

Reply:
originally I expected
```
          "/tutorial/" = {
            root = "${plutus.plutus-playground.documentation-site}/tutorial/";
          };
```
However that tries to use files `*/tutorial/tutorial/*`, you could spend time working out regexes to redirect things to the correct location but this seems much simpler.